src/index.html

```diff
@@ -4,9 +4,468 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo__img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a class="nav__item is-active" href="#apple">apple</a>
+          </li>
+
+          <li>
+            <a class="nav__item" href="#samsung">samsung</a>
+          </li>
+
+          <li>
+            <a class="nav__item" href="#smartphones">smartphones</a>
+          </li>
+
+          <li>
+            <a
+              class="nav__item"
+              href="#laptop&computers"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li>
+            <a class="nav__item" href="#gadgets">gadgets</a>
+          </li>
+
+          <li>
+            <a class="nav__item" href="#tablets">tablets</a>
+          </li>
+
+          <li>
+            <a class="nav__item" href="#photo">photo</a>
+          </li>
+
+          <li>
+            <a class="nav__item" href="#video">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page">
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image-imac"
+        >
+
+        <div class="card__product-description">
+          <h3 class="card__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">
+              Price:
+            </p>
+
+            <p class="card__price-numbers">
+              $2,199
+            </p>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card" data-qa="card">
```

Comment:
Consider will be better it you use the article tag here for the card
```suggestion
      <article class="card" data-qa="card">
```